SSA/Projects/InstCombine/Tactic.lean

```diff
@@ -12,26 +12,46 @@ open MLIR AST
 open Std (BitVec)
 open Ctxt
 
+-- TODO: Do we call this dialect the "Alive" dialect (as hinted by the `simp_alive_peephole` name),
+--       or the "InstCombine" dialect (as hinted by the `InstCombine` namespace that everything
+--       lives in), or the "LLVM" dialect (as we do in the paper, and the simp-sets,
+--       and the `LLVM` namespace)?
+
 open MLIR AST in
 /--
-- We first simplify `Com.refinement` to see the context `Γv`.
-- We `simp_peephole Γv` to simplify context accesses by variables.
-- We simplify the translation overhead.
-- Then we introduce variables, `cases` on the variables to eliminate the `none` cases.
-- We cannot leave it at this state, since then the variables will be inaccessible.
-- So, we revert the variables for the user to re-introduce them as they see fit.
--/
+`simp_alive_peephole` extends `simp_peephole` to simplify goals about refinement of `LLVM`
+programs into statements about just bitvectors.
+
+To wit, the tactic expects a goal of the form: `Com.Refinement com₁ com₂`
```

Comment:
I do not understand the expression `to wit`.

Reply:
I use 'To wit' to flag I'm making an earlier statement (here "goals about refinement of `LLVM` programs") more precise (by stating exactly the expression shape). Or, according to the [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/to-wit):
> used to make clearer or more particular something that you have already said

Admittedly, it's a bit of an obscure phrase, I